crates/turborepo-lib/src/cli.rs

```diff
@@ -250,6 +256,10 @@ pub enum Command {
         /// Do not create or modify .gitignore (default false)
         #[clap(long)]
         no_gitignore: bool,
+
+        /// Specify what should be linked (default "remote cache")
+        #[clap(long, value_enum)]
+        target: Option<LinkTarget>,
```

Comment:
Why not do both links in the same step instead of having two separate paths?

Reply:
Because you may want to link / unlink without always affecting both. Also during rollout this is easier option. 

We may want to change this in the future once we're fully rolled out